python-challenges/email-scheduler/devashmit-day39/scheduler.py

```
@@ -0,0 +1,46 @@
+def load_recipients(path):
+    ext=Path(path).suffix.lower()
+    if ext==".csv":
+        with open(path,newline="",encoding="utf-8") as f:return list(csv.DictReader(f))
+    with open(path,encoding="utf-8") as f:return json.load(f)
+
+def send_email(to,subject,body,dry_run):
+    if dry_run:print(f"  [DRY RUN] → {to}: {subject[:50]}");logging.info(f"DRY RUN to={to}");return
+    msg=MIMEMultipart();msg["Subject"]=subject;msg["From"]=f"{FROM_NAME} <{SMTP_USER}>";msg["To"]=to
+    msg.attach(MIMEText(body,"plain"))
+    try:
+        with smtplib.SMTP(SMTP_HOST,SMTP_PORT) as s:s.ehlo();s.starttls();s.login(SMTP_USER,SMTP_PASS);s.sendmail(SMTP_USER,to,msg.as_string())
```

Ruff (0.15.20)

[error] Multiple statements on one line (colon)

(E701)